modules/local/profile_dists/main.nf

```diff
@@ -31,9 +31,6 @@ process PROFILE_DISTS{
     if(columns){
         args = args + " --columns $columns"
     }
-    if(params.pd_force){
```

Comment:
Removed `--force` as a user-controlled parameter as it did not seem relevant for users to adjust.